tests/unit_tests/libs_tests/SFML_tests/Components/AnimationComponent_tests.cpp

```diff
@@ -0,0 +1,16 @@
+/*
+** EPITECH PROJECT, 2022
+** R-Type [WSL : Ubuntu]
+** File description:
+** AnimationComponent
+*/
+
+#include <criterion/criterion.h>
+#include "AnimationComponent.hpp"
```

Comment:
Is it the correct path ? 
Should not it be `GraphicECS/SFML/Components/...` ?

Reply:
It works